activerecord/lib/active_record/persistence.rb

```diff
@@ -222,10 +222,13 @@ def upsert(attributes, returning: nil, unique_by: nil)
       #   new records may considered duplicates according to different criteria. For MySQL,
       #   an upsert will take place if a new record violates _any_ unique constraint. For
       #   Postgres and SQLite, new rows will replace existing rows when the new row has the
-      #   same primary key as the existing row. By defining <tt>:unique_by</tt>, you can supply
-      #   a different key for matching new records to existing ones than the primary key.
+      #   same primary key as the existing row. In case of SQLite, an upsert operation causes
+      #   an insert to behave as an update or a no-op if the insert would violate
+      #   a uniqueness constraint. By defining <tt>:unique_by</tt>, you can supply
+      #   a different unique constraint for matching new records to existing ones than the
+      #   primary key.
```

Comment:
Can you change a line above to: `new records may be considered`. There is a missing `may`

Reply:
Done.